INSTALL.rst

```diff
@@ -53,29 +51,20 @@ an error message otherwise.
 Installation via pip and conda
 ------------------------------------------------------------------------------
 
-These install only ``scikit-image`` and its dependencies; pip has an option to
-include related packages.
-
 .. _install-via-pip:
 
 pip
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Prerequisites to a pip install: You're able to use your system's command line to
-install packages and are using a
+Prerequisites to a pip install: you must be able to use ``pip`` on
+your command line to install packages.
+
+We strongly recommend the use of a
 `virtual environment
-<https://towardsdatascience.com/virtual-environments-104c62d48c54?gi=2532aa12906#ee81>`_
-(any of
-`several
-<https://stackoverflow.com/questions/41573587/what-is-the-difference-between-venv-pyvenv-pyenv-virtualenv-virtualenvwrappe>`_\
-).
-
-While it is possible to use pip without a virtual environment, it is not advised:
-virtual environments create a clean Python environment that does not interfere
-with any existing system installation, can be easily removed, and contain only
-the package versions your application needs. They help avoid a common
-challenge known as
-`dependency hell <https://en.wikipedia.org/wiki/Dependency_hell>`_.
+<https://towardsdatascience.com/virtual-environments-104c62d48c54?gi=2532aa12906#ee81>`_.
+They create a clean Python environment that does not interfere
+with the existing system installation, can be easily removed, and contain only
```

Comment:
```suggestion
with your existing system installation, that can be easily removed, and that contains only
```

Reply:
I added the plural; I think the rest is OK as is.